controllers/invoice.ctrl.go

```diff
@@ -24,5 +28,23 @@ func (controller *InvoiceController) Invoice(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, responses.BadArgumentsError)
 	}
 
+	if controller.svc.Config.MaxVolume > 0 {
+		volume, err := controller.svc.GetVolumeOverPeriod(c.Request().Context(), user.ID, time.Duration(controller.svc.Config.MaxVolumePeriod*int64(time.Second)))
+		if err != nil {
+			c.Logger().Errorj(
+				log.JSON{
+					"message":        "error creating invoice",
+					"error":          err,
+					"lndhub_user_id": user.ID,
+				},
+			)
+			return c.JSON(http.StatusInternalServerError, responses.GeneralServerError)
+		}
+		if volume > controller.svc.Config.MaxVolume {
+			sentry.CaptureMessage(fmt.Sprintf("transaction volume exceeded for user %d", user.ID))
```

Comment:
should this error message include "creating invoice" so it's easier to understand where the error happened?

Reply:
The middleware should take care of this right?

Reply:
Ah ok, cool!